include/graph.h

```diff
@@ -111,6 +111,13 @@ class Graph {
   //! \param[in] path Edges of the path from source to destination
   //! \retval cost Cost of traversed path
   abm::graph::weight_t path_cost(const std::vector<graph::vertex_t>& path);
+  
+  //! Edge cost
```

Comment:
Please add this in a separate PR this is not part of the Agent class

Reply:
I added an issue for the moment